src/@next/components/organisms/BraintreePaymentGateway/BraintreePaymentGateway.tsx

```diff
@@ -87,9 +96,11 @@ const BraintreePaymentGateway: React.FC<IProps> = ({
         lastDigits: payment?.lastDigits,
       });
     } else {
-      setSubmitErrors([
-        { message: "Braintree client error occured during payment." },
-      ]);
+      const braintreePayloadErrors = [
+        { message: "Braintree returns no token in payload." },
```

Comment:
Is it right? It doesn't seem right as a sentence in this context

Reply:
Is this an error message displayed to the user?

Reply:
I have changed the message. @maarcingebala yes, it is - it shouldn't rather occur, but I think it's better to handle even such case, as token is required field to make payment on saleor.

Reply:
An error message like this - even if it happens rarely - it will eventually happen and the user will have no idea what is a token or what is a payload. I would consult such error messages with @pwgryglak as it is also part of UX.